kornia/nerf/nerf_solver.py

```diff
@@ -143,17 +144,17 @@ def _train_one_epoch(self) -> float:
         ray_dataset.init_ray_dataset(self._num_img_rays)
         ray_dataset.init_images_for_training(self._imgs)  # FIXME: Do we need to load the same images on each Epoch?
         ray_data_loader = instantiate_ray_dataloader(ray_dataset, self._batch_size, shuffle=True)
-        total_psnr = 0.0
+        total_psnr = torch.tensor(0.0, device=self._device, dtype=self._dtype)
         for i_batch, (origins, directions, rgbs) in enumerate(ray_data_loader):
             rgbs_model = self._nerf_model(origins, directions)
             loss = F.mse_loss(rgbs_model, rgbs)
 
-            total_psnr += psnr(rgbs_model, rgbs, 1.0)
+            total_psnr = psnr(rgbs_model, rgbs, 1.0) + total_psnr
```

Comment:
any idea why mypy doesn't like that?

Reply:
Nops, i don't understand why mypy is mad with operation between float and tensor here.